packages/legacy/package.json

```diff
@@ -59,7 +59,7 @@
     "@rest-hooks/test": "^2.0.0-beta.2"
   },
   "peerDependencies": {
-    "@rest-hooks/core": "^1.0.0-beta.0",
+    "@rest-hooks/core": "^1.0.0-delta.0",
```

Comment:
what are we, starting a sorority 😛

Reply:
haha. i realized that with so many packages with continual breaking changes along beta release that I needed to be able to distinguish breaking changes *within* a pre-release. going through greek letters allows me to control that.